capgains/transactions_reader.py

```diff
@@ -53,26 +54,26 @@ def get_transactions(cls, csv_file):
                     qty_idx = cls.columns.index("qty")
                     qty_str = entry[qty_idx]
                     try:
-                        entry[qty_idx] = int(qty_str)
-                    except ValueError:
```

Comment:
This is the main reason partial shares weren't supported right?

Reply:
Yeah that's right. We were truncating the decimal portion of the value by making it an integer, which lead to miscalculations later on.